Bottom: 552216c4c92cac1034077b23311e7374f0379b5d
Top:    fc0d4fef187ee5bca74ac50c3bb7806917a0a428
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-19 16:02:26 -0700

Refresh of dbg

---

diff --git a/src/preload.c b/src/preload.c
index e429ad6..47e1e64 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -491,6 +491,7 @@ static int connect_fork(int socket, const struct sockaddr *addr, socklen_t addrl
 		return ret;
 
 	real.close(fd);
+	printf("connect_fork - connecting rsocket");
 	return rconnect(ret, addr, addrlen);
 }
 
@@ -901,6 +902,7 @@ pid_t fork(void)
 	copysockopts(dfd, sfd, &rs, &real);
 	real.shutdown(sfd, SHUT_RDWR);
 	real.close(sfd);
+	printf("fork - using rsocket\n");
 	fd_store(last_accept, dfd, fd_rsocket);
 
 lclose:
